plugins/carla/cmake/carla-bridge-native.cmake

```diff
@@ -0,0 +1,56 @@
+add_executable(carla-bridge-native)
+mark_as_advanced(carla-bridge-native)
+
+target_compile_definitions(
+  carla-bridge-native
+  PRIVATE BUILDING_CARLA BUILD_BRIDGE BUILD_BRIDGE_ALTERNATIVE_ARCH CARLA_BACKEND_NAMESPACE=CarlaOBS
+          CARLA_LIB_EXT="${CMAKE_SHARED_LIBRARY_SUFFIX}" $<$<BOOL:${X11_FOUND}>:HAVE_X11>)
+
+target_compile_options(carla-bridge-native PRIVATE $<$<BOOL:${MSVC}>:/wd4244 /wd4267 /wd4273>
+                                                   $<$<NOT:$<BOOL:${MSVC}>>:-Wno-error -Werror=vla> ${X11_CFLAGS})
+
+target_include_directories(
+  carla-bridge-native
+  PRIVATE carla/source
+          carla/source/backend
+          carla/source/backend/engine
+          carla/source/backend/plugin
+          carla/source/includes
+          carla/source/modules
+          carla/source/utils
+          ${X11_INCLUDE_DIRS})
+
+target_link_directories(carla-bridge-native PRIVATE ${X11_LIBRARY_DIRS})
+
+target_link_libraries(carla-bridge-native PRIVATE carla::jackbridge carla::lilv carla::rtmempool carla::water
+                                                  ${X11_LIBRARIES})
+
+target_sources(
+  carla-bridge-native
+  PRIVATE carla/source/bridges-plugin/CarlaBridgePlugin.${CARLA_OBJCPP_EXT}
+          carla/source/backend/CarlaStandalone.${CARLA_OBJCPP_EXT}
+          carla/source/backend/engine/CarlaEngine.cpp
+          carla/source/backend/engine/CarlaEngineBridge.cpp
+          carla/source/backend/engine/CarlaEngineClient.cpp
+          carla/source/backend/engine/CarlaEngineData.cpp
+          carla/source/backend/engine/CarlaEngineInternal.cpp
+          carla/source/backend/engine/CarlaEnginePorts.cpp
+          carla/source/backend/engine/CarlaEngineRunner.cpp
+          carla/source/backend/plugin/CarlaPlugin.cpp
+          carla/source/backend/plugin/CarlaPluginBridge.cpp
+          carla/source/backend/plugin/CarlaPluginJuce.cpp
+          carla/source/backend/plugin/CarlaPluginInternal.cpp
+          carla/source/backend/plugin/CarlaPluginAU.cpp
+          carla/source/backend/plugin/CarlaPluginCLAP.${CARLA_OBJCPP_EXT}
+          carla/source/backend/plugin/CarlaPluginLADSPADSSI.cpp
+          carla/source/backend/plugin/CarlaPluginLV2.cpp
+          carla/source/backend/plugin/CarlaPluginVST2.${CARLA_OBJCPP_EXT}
+          carla/source/backend/plugin/CarlaPluginVST3.${CARLA_OBJCPP_EXT})
+
+set_target_properties(carla-bridge-native PROPERTIES FOLDER plugins OSX_ARCHITECTURES "x86_64;arm64")
```

Comment:
Prefer to use `set_property(TARGET <target> PROPERTY <property> <values>)` when adding multiple values (like `x86_64` and `arm64` in this case).

Use `APPEND PROPERTY` when actually adding new values in this list (because one architecture will always be in this list, dropping `APPEND` would actually be more correct for `OSX_ARCHITECTURES` in this case).

Reply:
no longer applies, marking as resolved (but the hint was quite welcome, applied on the carla side, thanks!)